### PR TITLE
Update the app Italian resource

### PR DESCRIPTION
**Pull request by marcoM32**

### CODE REVIEW

**Comment by jonan** [comment]
Please indent the file using spaces instead of tabs.

**Comment by marcoM32** [comment]
Because this translation has not yet been included

**Comment by jonan** [comment]
It's still reindenting all the file: https://github.com/jonan/ForkHub/pull/320/files

Compare the diff with this one: https://github.com/jonan/ForkHub/pull/324/files
Only the modified lines should appear in the diff.

**Comment by marcoM32** [comment]
Now the merge should be possible...

**Comment by marcoM32** [comment]
Please add my work on the official repo

File: app/src/main/res/values-it/strings.xml [hunk 1]
```diff
@@ -51,90 +57,139 @@
...
-    <string name="updating_labels">Aggiornamento Etichette…</string>
-    <string name="updating_milestone">Aggiornamento Milestone…</string>
+    <string name="updating_assignee">Aggiornamento Assegnatarioż/string>
+    <string name="updating_issue">Aggiornamento Segnalazioneż/string>
+    <string name="updating_labels">Aggiornamento Etichetteż/string>
+    <string name="updating_milestone">Aggiornamento Milestoneż/string>
+    <!--  -->
+
+    <!-- Issue events texts -->
+    <string name="issue_event_label_assigned">衳tata assegnata da %1$s</string>
```
Comment (jonan) [inline app/src/main/res/values-it/strings.xml]: example

File: app/src/main/res/values-it/strings.xml [hunk 2]
```diff
@@ -144,28 +199,41 @@
...
+    <string name="unstarring_gist">Rimozione del Gist dai preferiti in corsoż/string>
     <string name="accounts_label">Account</string>
     <string name="select_assignee">Seleziona Assegnatario</string>
     <string name="select_milestone">Seleziona Milestone</string>
     <string name="select_labels">Seleziona Etichette</string>
-    <string name="select_ref">Select Branch or Tag</string>
+    <string name="select_ref">Seleziona un Branch o un Tag</string>
+    <string name="login_pass_explanation">ForkHub usa la tua password solo per generare un &lt;i>Token di accesso personale&lt;/i>. Se preferisci puoi &lt;a href=\"https://github.com/settings/tokens\">crearlo tu stesso&lt;/a> e successivamente utilizzarlo qui come password.</string>
+    <string name="enter_otp_code_title">Codice di autenticazione</string>
+    <string name="enter_otp_code_message">L\'autenticazione a doppio step 衡bilitata per il tuo account. Inserisci il codice di autenticazione per verificare la tua identit஼/string>
```
Comment (jonan) [inline app/src/main/res/values-it/strings.xml]: example

File: app/src/main/res/values-it/strings.xml [hunk 3]
```diff
@@ -211,16 +288,17 @@
...
     <string name="reopen">Riapri</string>
     <string name="title_invalid_github_url">URL GitHub non valida</string>
     <string name="message_invalid_github_url">La applicazione non riesce ad aprire il seguente URL:\n{0}</string>
-    <string name="recently_viewed">VISUALIZZATI RECENTEMENTE</string>
     <string name="cancel">Annulla</string>
+    <string name="edited">(editato)</string>
     <string name="authenticator_conflict_title">Conflitto nell\'applicazione</string>
-    <string name="authenticator_conflict_message">E\' già stata configurata un\' altra applicazione per l\' autenticazione di GitHub.\n\nPer utilizzare l\'app di Github rimuovi l\'altra applicazione dagli Account e dalle impostazioni di sincronizzazione e disintallala.</string>
-    <string name="opening_repository">Apertura in corso {0}…</string>
+    <string name="authenticator_conflict_message">E\' giࡳtata configurata un\' altra applicazione per l\' autenticazione di GitHub.\n\nPer utilizzare l\'app di Github rimuovi l\'altra applicazione dagli Account e dalle impostazioni di sincronizzazione e disintallala.</string>
```
Comment (jonan) [inline app/src/main/res/values-it/strings.xml]: example

**Review by jonan — CHANGES_REQUESTED** [review]
You still have a few places with wrong characters.

I think you're using a invalid encoding.

**Review by newtires — APPROVED** [review]

**Review by Tawat99 — APPROVED** [review]

**Pull request closed without merging**